projects/core/src/user/store/selectors/user-replenishment-orders.selectors.spec.ts

```diff
@@ -0,0 +1,114 @@
+import { TestBed } from '@angular/core/testing';
+import { select, Store, StoreModule } from '@ngrx/store';
+import { ReplenishmentOrderList } from '../../../model/replenishment-order.model';
+import { StateUtils } from '../../../state/utils/index';
+import { UserActions } from '../actions/index';
+import * as fromReducers from '../reducers/index';
+import { UsersSelectors } from '../selectors/index';
+import { StateWithUser, USER_FEATURE } from '../user-state';
+
+const mockReplenishmentOrderList: ReplenishmentOrderList = {
+  replenishmentOrders: [],
+  pagination: {
+    currentPage: 1,
+    pageSize: 5,
+  },
+  sorts: [{ code: 'byPage' }],
+};
+
```

Comment:
assuming some parts were based on repl order details, you may have noticed, you need loading state as well